### PR TITLE
.github/pr-check-license.yaml: fix the license check issue

### PR DESCRIPTION
**Pull request by leyao-daily**
This PR is to quickly fix the license check issue

### CODE REVIEW

**Comment by leyao-daily** [comment]
@dongx1x I move your license fix to this PR to avoid blocking other PRs scan.

**Review by Hulongyin — APPROVED** [review]
LGTM. Thanks

File: .github/workflows/pr-license-check.yaml [hunk 1]
```diff
@@ -18,11 +18,14 @@ jobs:
       - name: Check License
         run: |
           set -ex
+          python3 -m pip install --upgrade pip
           python3 -m pip install liccheck
+
           license_files=$(find -type f -name "requirements.txt")
```
Comment (hwang37) [inline .github/workflows/pr-license-check.yaml:24]: I don't find "requirements.txt" file in folder.
1) Do we still need this .txt file?
2) Or do we need to create a such file?

Comment (leyao-daily) [reply .github/workflows/pr-license-check.yaml:24]: It is under the folder cnap.

**Pull request merged**